<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>groupBox_show</class>
 <widget class="QGroupBox" name="groupBox_show">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>473</width>
    <height>392</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>GroupBox</string>
  </property>
  <property name="title">
   <string>Алгоритм</string>
  </property>
  <property name="checkable">
   <bool>true</bool>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="7" column="1">
    <widget class="QDoubleSpinBox" name="doubleSpinBox_Dr">
     <property name="decimals">
      <number>4</number>
     </property>
     <property name="minimum">
      <double>-10000.000000000000000</double>
     </property>
     <property name="maximum">
      <double>10000.000000000000000</double>
     </property>
     <property name="singleStep">
      <double>0.001000000000000</double>
     </property>
     <property name="value">
      <double>30.000000000000000</double>
     </property>
    </widget>
   </item>
   <item row="0" column="0">
    <widget class="QLabel" name="label_3">
     <property name="text">
      <string>DELTA:</string>
     </property>
    </widget>
   </item>
   <item row="0" column="1">
    <widget class="QDoubleSpinBox" name="doubleSpinBox_DELTA">
     <property name="decimals">
      <number>4</number>
     </property>
     <property name="minimum">
      <double>-10000.000000000000000</double>
     </property>
     <property name="maximum">
      <double>10000.000000000000000</double>
     </property>
     <property name="singleStep">
      <double>0.001000000000000</double>
     </property>
     <property name="value">
      <double>70.000000000000000</double>
     </property>
    </widget>
   </item>
   <item row="1" column="0">
    <widget class="QLabel" name="label_12">
     <property name="text">
      <string>WALL_ANALYSIS_DX_GAP:</string>
     </property>
    </widget>
   </item>
   <item row="1" column="1">
    <widget class="QDoubleSpinBox" name="doubleSpinBox_WALL_ANALYSIS_DX_GAP">
     <property name="decimals">
      <number>4</number>
     </property>
     <property name="minimum">
      <double>-10000.000000000000000</double>
     </property>
     <property name="maximum">
      <double>10000.000000000000000</double>
     </property>
     <property name="singleStep">
      <double>0.001000000000000</double>
     </property>
     <property name="value">
      <double>15.000000000000000</double>
     </property>
    </widget>
   </item>
   <item row="2" column="0">
    <widget class="QLabel" name="label_13">
     <property name="text">
      <string>WALL_ANALYSIS_ANGLE:</string>
     </property>
    </widget>
   </item>
   <item row="2" column="1">
    <widget class="QDoubleSpinBox" name="doubleSpinBox_WALL_ANALYSIS_ANGLE">
     <property name="decimals">
      <number>4</number>
     </property>
     <property name="minimum">
      <double>-10000.000000000000000</double>
     </property>
     <property name="maximum">
      <double>10000.000000000000000</double>
     </property>
     <property name="singleStep">
      <double>0.001000000000000</double>
     </property>
     <property name="value">
      <double>90.000000000000000</double>
     </property>
    </widget>
   </item>
   <item row="3" column="0">
    <widget class="QLabel" name="label_14">
     <property name="text">
      <string>MIN_LIDAR_DISTANCE:</string>
     </property>
    </widget>
   </item>
   <item row="3" column="1">
    <widget class="QDoubleSpinBox" name="doubleSpinBox_MIN_LIDAR_DISTANCE">
     <property name="decimals">
      <number>4</number>
     </property>
     <property name="minimum">
      <double>-10000.000000000000000</double>
     </property>
     <property name="maximum">
      <double>10000.000000000000000</double>
     </property>
     <property name="singleStep">
      <double>0.001000000000000</double>
     </property>
     <property name="value">
      <double>20.000000000000000</double>
     </property>
    </widget>
   </item>
   <item row="4" column="0">
    <widget class="QLabel" name="label_15">
     <property name="text">
      <string>MAX_LIDAR_DISTANCE:</string>
     </property>
    </widget>
   </item>
   <item row="4" column="1">
    <widget class="QDoubleSpinBox" name="doubleSpinBox_MAX_LIDAR_DISTANCE">
     <property name="decimals">
      <number>4</number>
     </property>
     <property name="minimum">
      <double>-10000.000000000000000</double>
     </property>
     <property name="maximum">
      <double>10000.000000000000000</double>
     </property>
     <property name="singleStep">
      <double>0.001000000000000</double>
     </property>
     <property name="value">
      <double>600.000000000000000</double>
     </property>
    </widget>
   </item>
   <item row="5" column="0">
    <widget class="QLabel" name="label_16">
     <property name="text">
      <string>TRUSTABLE_LIDAR_DISTANCE:</string>
     </property>
    </widget>
   </item>
   <item row="5" column="1">
    <widget class="QDoubleSpinBox" name="doubleSpinBox_TRUSTABLE_LIDAR_DISTANCE">
     <property name="decimals">
      <number>4</number>
     </property>
     <property name="minimum">
      <double>-10000.000000000000000</double>
     </property>
     <property name="maximum">
      <double>10000.000000000000000</double>
     </property>
     <property name="singleStep">
      <double>0.001000000000000</double>
     </property>
     <property name="value">
      <double>400.000000000000000</double>
     </property>
    </widget>
   </item>
   <item row="6" column="0">
    <widget class="QLabel" name="label_17">
     <property name="text">
      <string>MIN_GAP_WIDTH:</string>
     </property>
    </widget>
   </item>
   <item row="6" column="1">
    <widget class="QDoubleSpinBox" name="doubleSpinBox_MIN_GAP_WIDTH">
     <property name="decimals">
      <number>4</number>
     </property>
     <property name="minimum">
      <double>-10000.000000000000000</double>
     </property>
     <property name="maximum">
      <double>10000.000000000000000</double>
     </property>
     <property name="singleStep">
      <double>0.001000000000000</double>
     </property>
     <property name="value">
      <double>50.000000000000000</double>
     </property>
    </widget>
   </item>
   <item row="7" column="0">
    <widget class="QLabel" name="label_18">
     <property name="text">
      <string>Dr:</string>
     </property>
    </widget>
   </item>
   <item row="8" column="0">
    <widget class="QLabel" name="label_19">
     <property name="text">
      <string>Da:</string>
     </property>
    </widget>
   </item>
   <item row="8" column="1">
    <widget class="QDoubleSpinBox" name="doubleSpinBox_Da">
     <property name="decimals">
      <number>4</number>
     </property>
     <property name="minimum">
      <double>-10000.000000000000000</double>
     </property>
     <property name="maximum">
      <double>10000.000000000000000</double>
     </property>
     <property name="singleStep">
      <double>0.001000000000000</double>
     </property>
     <property name="value">
      <double>0.100000000000000</double>
     </property>
    </widget>
   </item>
   <item row="9" column="0">
    <widget class="QLabel" name="label_21">
     <property name="text">
      <string>Dh:</string>
     </property>
    </widget>
   </item>
   <item row="9" column="1">
    <widget class="QDoubleSpinBox" name="doubleSpinBox_Dh">
     <property name="decimals">
      <number>4</number>
     </property>
     <property name="minimum">
      <double>-10000.000000000000000</double>
     </property>
     <property name="maximum">
      <double>10000.000000000000000</double>
     </property>
     <property name="singleStep">
      <double>0.001000000000000</double>
     </property>
     <property name="value">
      <double>0.700000000000000</double>
     </property>
    </widget>
   </item>
   <item row="11" column="0">
    <spacer name="verticalSpacer_2">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>96</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="10" column="0">
    <widget class="QLabel" name="label_22">
     <property name="text">
      <string>MIN_GAP_ANGLE:</string>
     </property>
    </widget>
   </item>
   <item row="10" column="1">
    <widget class="QDoubleSpinBox" name="doubleSpinBox_MIN_GAP_ANGLE">
     <property name="decimals">
      <number>4</number>
     </property>
     <property name="minimum">
      <double>-10000.000000000000000</double>
     </property>
     <property name="maximum">
      <double>10000.000000000000000</double>
     </property>
     <property name="singleStep">
      <double>0.001000000000000</double>
     </property>
     <property name="value">
      <double>0.062800000000000</double>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
